### DISCUSSION TITLE
web3Modal.openModal not working

### DISCUSSION THREAD

**Discussion by 0xtchandragupta** [WalletConnect/walletconnect-monorepo#3200] [Developer Support]
Hi,
I am using the CDN based version of web3Modal as described here.
https://docs.walletconnect.com/2.0/web/web3modal/html/wagmi/installation#add-packages-cdn

I am using web3modal.openModal() to open the wallet connect modal to choose a wallet.
This was working fine when I checked a few days ago. 
But while checking today, 
web3modal.openModal() is just not doing anything.
It does not even throw an error on console or anything.
Not sure what has gone wrong.

I see that Metamask UI is now an updated one. Has that introduced some change due to which this is failing?

**Comment by glitch-txs** [comment]
Hi, do you have a public repo or any way we can reproduce this issue?

**Reply by 0xtchandragupta** [reply]
@glitch-txs 
It is live on our site. But you will need to sign in.
Will you be able to sign up?
After sign up navigate to Wallet tab and click the Link An Account button.

**Reply by glitch-txs** [reply]
I mean to look at the code and be able to run it*

**Reply by 0xtchandragupta** [reply]
@glitch-txs
I've exposed the web3modal object in the window object of the browser.
So if you're unable to sign up, you can just try executing
await window.web3modal.openModal() on the console on home page without signing in.
